paper.ipynb

```diff
@@ -129,23 +129,23 @@
    "id": "12164240",
```

Comment:
<p>Author didn't synchronize with the last version of the article as there are the old mention of figure labelling</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/jdh-observer/jnkqqTTKW8km/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Reply:
<p>fixed</p>